### PR TITLE
RTL: Flip some dashicons that have directionality

### PR DESCRIPTION
**Pull request by yoavf**
## Description
- `editor-outdent`, `editor-indent`, `list-view`, and `editor-ul` can and should be flipped in RTL.
- `editor-ol` (not taken care of yet) needs an RTL version: https://github.com/WordPress/dashicons/issues/250

## How Has This Been Tested?
Manual testing in LTR and RTL mode

## Screenshots (jpeg or gifs if applicable):
**Before:**
before-list
before-blocks

**After**
after-list
after-blocks

## Types of changes
CSS that will affect the RTL version, and will flip the icons.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: components/panel/style.scss [hunk 1]
```diff
@@ -133,3 +133,14 @@
 .components-panel .circle-picker {
 	padding-bottom: 20px;
 }
+
+body.rtl .gutenberg .dashicon {
```
Comment (mtias) [inline components/panel/style.scss]: this should probably go in a dashicons scss: https://github.com/WordPress/gutenberg/tree/master/components/dashicon

File: components/dashicon/index.js [hunk 2]
```diff
@@ -5,6 +5,11 @@ DO NOT EDIT THAT FILE! EDIT index-header.jsx and index-footer.jsx instead
 OR if you're looking to change now SVGs get output, you'll need to edit strings in the Gruntfile :)
 !!! */
 
+/**
+ * Internal dependencies
+ */
+import './style.scss';
```
Comment (aduth) [inline components/dashicon/index.js]: This file is generated, so this import would be overwritten next time the Dashicons is updated. It seems like the RTL flipping could be valuable to have in the base Dashicons repository, but in keeping it generic we'd not want to assume the presence of an `.rtl` body class. Would it be reasonable to detect RTL from JavaScript inside the component class and apply an inline style?

https://github.com/WordPress/dashicons/blob/master/sources/react/index-header.jsx
https://github.com/WordPress/dashicons/blob/master/sources/react/index-footer.jsx

File: components/dashicon/style.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+body.rtl .gutenberg .dashicon {
+  &.dashicons-editor-outdent,
```
Comment (aduth) [inline components/dashicon/style.scss]: Mixed whitespace, tabs and spaces.

**Review by aduth — CHANGES_REQUESTED** [review]

**Comment by gziolo** [comment]
@yoavf do you plan to finish this PR or should someone else take it over?

**Comment by azaozz** [comment]
May be just me but... https://user-images.githubusercontent.com/844866/33838336-f52ade10-de97-11e7-9d82-9448dc707c13.png looks odd with the numbered list icon in LTR and the bulleted one in RTL. Can we use "proper" RTL icons there? Then we only will need to swap the icons in the CSS, not "mirror" them.

**Comment by aduth** [comment]
Closing as stale without response.

**Pull request closed without merging**